x/deployment/types/v1beta2/resource_list_validation.go

```diff
@@ -129,6 +129,10 @@ func validateStorage(u types.Volumes) ([]sdk.Int, error) {
 		return nil, errors.Errorf("error: invalid unit storage, cannot be nil")
 	}
 
+	if len(u) > validationConfig.MaxGroupVolumes {
```

Comment:
Won't changing this require a chain upgrade?

Reply:
should not as we don't have deployments on both mainnet/testnet with storage.

Reply:
but once we do, it will, right?

Reply:
correct

Reply:
Let's not do that.  I don't want to get into another situation like we were in with the resource limits.

Reply:
actually not, it does not change message structure but limits amount of entries

Reply:
what if we want to change the limit

Reply:
well, yeah, it's gonna be nasty as software upgrade to the network will be required.
i'll make it as limit on the provider side, and print warning as you suggested above

Reply:
Are we just going to put a soft limit in on the provider where it doesn't bid on anything with less than a configured amount of volumes?

Reply:
yeah